configuration/configuration.go

```diff
@@ -144,6 +144,12 @@ func ParseArgs(args []string, configuration Configuration) (command string, para
 			newConfiguration.StartCreate = true
 		case "--delete-remote-wip-branch":
 			newConfiguration.ResetDeleteRemoteWipBranch = true
+		case "--room":
+			if i+1 != len(args) {
+				newConfiguration.TimerRoom = args[i+1]
+			}
```

Comment:
This block now has 3 duplications --branch as well as --message

Reply:
The following function could solve this issue:
```go
func SetNextArg(args []string, i *int, setValue func(value string)) {
	if *i+1 < len(args) {
		setValue(args[*i+1])
		*i += 1
	}
}
```

We could use it in the code as follows:
```go
SetNextArg(args, &i, func(value string) {
	newConfiguration.TimerRoom = value
})
```

This solution is in my opinion much more complex and therefore i would stay with 3 code duplications here.